### PR TITLE
MAINTAINERS: update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by nbusseneau**
- Add new committers
- Fix missing accent in Beatriz' name
- Fix Michal's name and GitHub handle

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -80,12 +82,14 @@ project.
 [John Fastabend]: https://github.com/jrfastab
 [Kornilios Kourtis]: https://github.com/kkourt
 [Laurent Bernaille]: https://github.com/lbernail
+[Louis De Los Santos]: https://github.com/ldelossa
 [Maciej Kwiek]: https://github.com/nebril
 [Martynas Pumputis]: https://github.com/brb
 [Michael Rostecki]: https://github.com/mrostecki
```
Comment (tklauser) [inline MAINTAINERS.md]: While at it, could you maybe also adjust Michal's GitHub name? /cc @vadorovsky

```suggestion
[Michael Rostecki]: https://github.com/vadorovsky
```

Comment (nbusseneau) [reply MAINTAINERS.md]: Sure!

Comment (nbusseneau) [reply MAINTAINERS.md]: Should I also update name from Michael to Michal?

Comment (tklauser) [reply MAINTAINERS.md]: Yeah, looks like the name wasn't properly spelled in `MAINTAINERS` from the beginning.

**Review by tklauser — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Review by christarazi — APPROVED** [review]

**Review by b3a-dev — APPROVED** [review]
LGTM

**Review by joestringer — APPROVED** [review]
:tada:

File: MAINTAINERS.md [hunk 2]
```diff
@@ -80,12 +82,14 @@ project.
 [John Fastabend]: https://github.com/jrfastab
 [Kornilios Kourtis]: https://github.com/kkourt
 [Laurent Bernaille]: https://github.com/lbernail
+[Louis De Los Santos]: https://github.com/ldelossa
```
Comment (ldelossa) [inline MAINTAINERS.md]: I sign off everything without spaces in the last name. "DeLosSantos"

**Review by ldelossa — CHANGES_REQUESTED** [review]

File: MAINTAINERS.md (hunk 2 above)
Comment (nbusseneau) [reply MAINTAINERS.md]: Was unsure which to use between the GitHub and Slack versions, and picked Slack's 🙈

Comment (ldelossa) [reply MAINTAINERS.md]: No worries :)

**Review by ldelossa — APPROVED** [review]

**Pull request merged**